pytest_fxa/plugin.py

```diff
@@ -28,14 +28,35 @@ def pytest_addoption(parser):
     )
 
 
-@pytest.fixture(scope='session')
+def pytest_configure(config):
+    config.addinivalue_line(
+        'markers',
+        'fxa_env(name,): mark tests to run against named Firefox Accounts '
+        'environment(s): ' + ', '.join(ENVIRONMENT_URLS.keys()))
+
+
+def pytest_generate_tests(metafunc):
+    if 'fxa_urls' not in metafunc.fixturenames:
+        return
+
+    envs = set([
+        env_name
+        for marker in metafunc.definition.iter_markers('fxa_env')
+        for env_name in marker.args])
+
+    if envs:
+        metafunc.parametrize('fxa_urls', envs, indirect=True)
```

Comment:
What happens if the ``fxa_env`` marker is applied without any arguments?

I assume the corresponding test would error, since ``fxa_urls`` doesn't receive a ``request`` object?! 🤔

Reply:
No error, we default to staging. I'll add a test to make that clearer.

Reply:
Ahh got it! 👍